### PR TITLE
Add second floor

### PR DESCRIPTION
**Pull request by klas3**

### CODE REVIEW

File: secondFloor.html [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+                <a-entity id="sceneHallRight2" class="scene" visible="false">
+                    <a-sky src="#hallRight2"></a-sky>
+                    <a-entity template="src: #link" data-src="sceneHallRight3" data-thumb="#arrow" position="-10 -2.3 -0.6" rotation="-63.5 90 0"></a-entity>
+                    <a-entity template="src: #link" data-src="sceneAudience221" data-thumb="#arrow" position="-0.5 -3.2 -1.9" rotation="-40 -60 40"></a-entity>
+                    <a-entity template="src: #link" data-src="sceneHallRight1" data-thumb="#arrow"position="10.2 -3.1 1" rotation="10 -105 0"></a-entity>
+                </a-entity>
+
+                <a-entity id="sceneAudience221" class="scene" visible="false">
+                    <a-sky src="#audience221"></a-sky>
+                    <a-entity template="src: #link" data-src="sceneHallRight2" data-thumb="#arrow" position="-11.8 -2.8 9.1" rotation="-70 170 -40"></a-entity>
```
Comment (Andreybest) [inline secondFloor.html]: Hard to click for mobile users

File: secondFloor.html [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+                </a-entity>
+
+                <a-entity id="sceneAudience217" class="scene" visible="false">
+                    <a-sky src="#audience217"></a-sky>
+                    <a-entity template="src: #link" data-src="sceneHallRight1" data-thumb="#arrow" position="8.2 -2.7 0.6" rotation="0.2 -80 -5"></a-entity>
+                </a-entity>
+
+                <a-entity id="sceneAudience218" class="scene" visible="false">
+                    <a-sky src="#audience218"></a-sky>
+                    <a-entity template="src: #link" data-src="sceneHallRight1" data-thumb="#arrow" position="-0.7 -4.7 11" rotation="-90 80 92"></a-entity>
```
Comment (Andreybest) [inline secondFloor.html]: Hard to click for mobile users

File: secondFloor.html [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+                    <a-sky src="#hallRight1"></a-sky>
+                    <a-entity template="src: #link" data-src="sceneHallRight2" data-thumb="#arrow" position="-0.2 -3.5 8" rotation="-70 170 0"></a-entity>
+                    <a-entity template="src: #link" data-src="sceneAudience217" data-thumb="#arrow" position="4.5 -4.3 -3.3" rotation="0.2 -50 10"></a-entity>
+                    <a-entity template="src: #link" data-src="sceneAudience218" data-thumb="#arrow" position="-5.2 -3.2 -2.4" rotation="0.2 60 -25"></a-entity>
+                    <a-entity template="src: #link" data-src="sceneHallMiddle1" data-thumb="#arrow" position="10.3 -3.2 0.5" rotation="10 -80 0"></a-entity>
+                </a-entity>
+
+                <a-entity id="sceneAudience217" class="scene" visible="false">
+                    <a-sky src="#audience217"></a-sky>
+                    <a-entity template="src: #link" data-src="sceneHallRight1" data-thumb="#arrow" position="8.2 -2.7 0.6" rotation="0.2 -80 -5"></a-entity>
```
Comment (Andreybest) [inline secondFloor.html]: Needs small remove

File: secondFloor.html [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+                <a-entity id="sceneAudience201Main" class="scene" visible="false">
+                    <a-sky src="#audience201Main"></a-sky>
+                    <a-entity template="src: #link" data-src="sceneAudience201Hall2" data-thumb="#arrow" position="4.2 -6.2 -10" rotation="-110 -180 180"></a-entity>
+                </a-entity>
+
+            <!--Right side-->
+                <a-entity id="sceneHallRight1" class="scene" visible="false">
+                    <a-sky src="#hallRight1"></a-sky>
+                    <a-entity template="src: #link" data-src="sceneHallRight2" data-thumb="#arrow" position="-0.2 -3.5 8" rotation="-70 170 0"></a-entity>
+                    <a-entity template="src: #link" data-src="sceneAudience217" data-thumb="#arrow" position="4.5 -4.3 -3.3" rotation="0.2 -50 10"></a-entity>
```
Comment (Andreybest) [inline secondFloor.html]: Needs small remove

File: secondFloor.html [hunk 5]
```diff
@@ -0,0 +1,237 @@
...
+                </script>
+            </a-assets>
+
+            <!--Middle-->
+                <a-entity id="sceneHallMiddle1" class="scene">
+                    <a-sky src="#hallMiddle1"></a-sky>
+                    <a-entity template="src: #link" data-src="sceneHallMiddle2" data-thumb="#arrow" position="-1 -5.2 10.1" rotation="-30 180 0"></a-entity>
+                    <a-entity template="src: #link" data-src="sceneHallRight1" data-thumb="#arrow" position="0.2 -4.7 -9.7" rotation="-60 0 -5"></a-entity>
+                    <a-entity template="src: #link" data-src="sceneDepartment" data-thumb="#arrow" position="-5.2 -4.4 -1.4" rotation="0 70 5"></a-entity>
+                    <a-entity template="src: #link" data-src="sceneAudience213" data-thumb="#arrow" position="4.2 -5 11.3" rotation="-80 -170 80"></a-entity>
```
Comment (Andreybest) [inline secondFloor.html]: Hard to click for mobile users

File: secondFloor.html [hunk 6]
```diff
@@ -0,0 +1,237 @@
...
+                </a-entity>
+
+                <a-entity id="sceneDepartment" class="scene" visible="false">
+                    <a-sky src="#department"></a-sky>
+                    <a-entity template="src: #link" data-src="sceneHallMiddle1" data-thumb="#arrow" position="8.2 -3.7 -3.3" rotation="0.2 -80 -5"></a-entity>
+                </a-entity>
+
+                <a-entity id="sceneAudience213" class="scene" visible="false">
+                    <a-sky src="#audience213"></a-sky>
+                    <a-entity template="src: #link" data-src="sceneHallMiddle1" data-thumb="#arrow" position="15.3 -3.7 2.8" rotation="0.2 -100 5"></a-entity>
```
Comment (Andreybest) [inline secondFloor.html]: Needs small remove

File: secondFloor.html [hunk 7]
```diff
@@ -0,0 +1,237 @@
...
+                    event-set__mouseleave="scale: 1 1 1"
+                    hotspot-clicked="loadScene: ${src}"
+                    sound="on: click; src: #click-sound"></a-entity>
+                </script>
+            </a-assets>
+
+            <!--Middle-->
+                <a-entity id="sceneHallMiddle1" class="scene">
+                    <a-sky src="#hallMiddle1"></a-sky>
+                    <a-entity template="src: #link" data-src="sceneHallMiddle2" data-thumb="#arrow" position="-1 -5.2 10.1" rotation="-30 180 0"></a-entity>
```
Comment (Andreybest) [inline secondFloor.html]: Needs small remove

File: secondFloor.html [hunk 8]
```diff
@@ -0,0 +1,237 @@
...
+                </a-entity>
+
+                <a-entity id="sceneLibrary" class="scene" visible="false">
+                    <a-sky src="#library"></a-sky>
+                    <a-entity template="src: #link" data-src="sceneHallMiddle2" data-thumb="#arrow" position="10.6 -2.7 9.8" rotation="-70 -70 0"></a-entity>
+                </a-entity>
+
+                <a-entity id="sceneAudience211" class="scene" visible="false">
+                    <a-sky src="#audience211"></a-sky>
+                    <a-entity template="src: #link" data-src="sceneHallMiddle2" data-thumb="#arrow" position="6.1 -3.8 -13.4" rotation="40 -30 25"></a-entity>
```
Comment (Andreybest) [inline secondFloor.html]: Hard to click for mobile users

File: secondFloor.html [hunk 9]
```diff
@@ -0,0 +1,237 @@
...
+                    <a-sky src="#hallMiddle2"></a-sky>
+                    <a-entity template="src: #link" data-src="sceneHallLeft1" data-thumb="#arrow" position="-9.8 -2.3 0" rotation="-60 90 0"></a-entity>
+                    <a-entity template="src: #link" data-src="sceneLibrary" data-thumb="#arrow" position="-2.7 -3.6 -8.4" rotation="10 30 -20"></a-entity>
+                    <a-entity template="src: #link" data-src="sceneAudience211" data-thumb="#arrow" position="4 -4.1 3.6" rotation="-110 30 180"></a-entity>
+                    <a-entity template="src: #link" data-src="sceneHallMiddle1" data-thumb="#arrow" position="10.3 -3.2 -1.9" rotation="10 -90 6"></a-entity>
+                </a-entity>
+
+                <a-entity id="sceneLibrary" class="scene" visible="false">
+                    <a-sky src="#library"></a-sky>
+                    <a-entity template="src: #link" data-src="sceneHallMiddle2" data-thumb="#arrow" position="10.6 -2.7 9.8" rotation="-70 -70 0"></a-entity>
```
Comment (Andreybest) [inline secondFloor.html]: Hard to click for mobile users

File: secondFloor.html [hunk 10]
```diff
@@ -0,0 +1,237 @@
...
+                <a-entity id="sceneHallLeft2" class="scene" visible="false">
+                    <a-sky src="#hallLeft2"></a-sky>
+                    <a-entity template="src: #link" data-src="sceneHallLeft3" data-thumb="#arrow" position="-12.5 -2.3 0" rotation="-60 90 0"></a-entity>
+                    <a-entity template="src: #link" data-src="sceneAudience209" data-thumb="#arrow" position="0.9 -4.9 4.4" rotation="-110 10.7 180"></a-entity>
+                    <a-entity template="src: #link" data-src="sceneHallLeft1" data-thumb="#arrow" position="10.3 -3.2 -2.8" rotation="10 -80 3"></a-entity>
+                </a-entity>
+
+                <a-entity id="sceneAudience209" class="scene" visible="false">
+                    <a-sky src="#audience209"></a-sky>
+                    <a-entity template="src: #link" data-src="sceneHallLeft2" data-thumb="#arrow" position="-8.2 -4.1 -9.2" rotation="-110 -130 180"></a-entity>
```
Comment (Andreybest) [inline secondFloor.html]: Hard to click for mobile users

File: secondFloor.html [hunk 11]
```diff
@@ -0,0 +1,237 @@
...
+                    <a-sky src="#hallLeft3"></a-sky>
+                    <a-entity template="src: #link" data-src="sceneHallLeft4" data-thumb="#arrow" position="-0.3 -2.6 -7.4" rotation="-40 0 0"></a-entity>
+                    <a-entity template="src: #link" data-src="sceneAudience207" data-thumb="#arrow" position="6.5 -4.1 4.7" rotation="-102 0 180"></a-entity>
+                    <a-entity template="src: #link" data-src="sceneAudience205" data-thumb="#arrow" position="-5.2 -4.9 6.3" rotation="-110 6 180"></a-entity>
+                    <a-entity template="src: #link" data-src="sceneHallLeft2" data-thumb="#arrow" position="10.3 -3.2 -0.2" rotation="10 -80 0"></a-entity>
+                </a-entity>
+
+                <a-entity id="sceneAudience207" class="scene" visible="false">
+                    <a-sky src="#audience207"></a-sky>
+                    <a-entity template="src: #link" data-src="sceneHallLeft3" data-thumb="#arrow" position="-8.1 -4.1 -14.1" rotation="-110 -188 180"></a-entity>
```
Comment (Andreybest) [inline secondFloor.html]: Hard to click for mobile users

File: secondFloor.html [hunk 12]
```diff
@@ -0,0 +1,237 @@
...
+                <a-entity id="sceneHallLeft4" class="scene" visible="false">
+                    <a-sky src="#hallLeft4"></a-sky>
+                    <a-entity template="src: #link" data-src="sceneAudience201Hall1" data-thumb="#arrow" position="-10 -5 0" rotation="-110 -93 180"></a-entity>
+                    <a-entity template="src: #link" data-src="sceneAudience203" data-thumb="#arrow" position="0 -4.6 4" rotation="-80 186.3 0"></a-entity>
+                    <a-entity template="src: #link" data-src="sceneHallLeft3" data-thumb="#arrow" position="10.3 -3.2 -2.7" rotation="20 -80 5"></a-entity>
+                </a-entity>
+
+                <a-entity id="sceneAudience203" class="scene" visible="false">
+                    <a-sky src="#audience203"></a-sky>
+                    <a-entity template="src: #link" data-src="sceneHallLeft4" data-thumb="#arrow" position="-8 -4.1 -10.4" rotation="-103.2 -188 180"></a-entity>
```
Comment (Andreybest) [inline secondFloor.html]: Hard to click for mobile users

File: secondFloor.html [hunk 13]
```diff
@@ -0,0 +1,237 @@
...
+                </a-entity>
+
+                <a-entity id="sceneAudience205" class="scene" visible="false">
+                    <a-sky src="#audience205"></a-sky>
+                    <a-entity template="src: #link" data-src="sceneHallLeft3" data-thumb="#arrow" position="1.6 -4.1 -10.6" rotation="-110 -180 180"></a-entity>
+                </a-entity>
+
+                <a-entity id="sceneHallLeft4" class="scene" visible="false">
+                    <a-sky src="#hallLeft4"></a-sky>
+                    <a-entity template="src: #link" data-src="sceneAudience201Hall1" data-thumb="#arrow" position="-10 -5 0" rotation="-110 -93 180"></a-entity>
```
Comment (Andreybest) [inline secondFloor.html]: Needs small remove

**Review by Andreybest — CHANGES_REQUESTED** [review]
Everything is good, but many arrows need repositioning.
And 'director'sOffice.jpg' should be renamed to 'directorsWaitingRoom.jpg'.

**Comment by Andreybest** [comment]
Also forgot to mention, index.html needs a link to 2 floor.

**Comment by klas3** [comment]
Problematic arrows has been fixed, also all floors-connecting links has been added.

File: index.html [hunk 14]
```diff
@@ -17,7 +17,9 @@
 				<audio id="click-sound" crossorigin="anonymous" src="https://cdn.aframe.io/360-image-gallery-boilerplate/audio/click.ogg"></audio>
 				<img id="arrow" crossorigin="anonymous" src="img/navArrow.png" transparent="true" alpha-test="0.8">
 				<img id="logo" crossorigin="anonymous" src="img/logo.png">
+				<img id="secondFloorUpStairs" crossorigin="anonymous" src="img/floorsIcons/secondFloorUpStairs.jpg">
 				<img id="thirdFloorUpStairs" crossorigin="anonymous" src="img/floorsIcons/thirdFloorUpStairs.jpg">
+				<img id="secondFloorUpStairs" crossorigin="anonymous" src="img/floorsIcons/secondFloorUpStairs.jpg">
```
Comment (Andreybest) [inline index.html:21]: Remove duplicating links

File: index.html [hunk 15]
```diff
@@ -57,7 +59,9 @@
 				<a-entity id="sceneMenuStairs" class="scene" visible="false">
 					<a-sky src="#menuStairs"></a-sky>
 					<a-entity template="src: #link" data-src="sceneMenuLobby" data-thumb="#arrow" position="15.000 -7.000 1.500" rotation="-120.000 80.000 180.000" scale="1.500 1.500 0.000"></a-entity>
+					<a-entity link="href: secondFloor.html;" template="src: #link" data-src="" data-thumb="#secondFloorUpStairs" position="6.000 2.000 -10.000" rotation="0.000 0 0.000" scale="1.500 1.500 0.000"></a-entity>
 					<a-entity link="href: thirdFloor.html;" template="src: #link" data-src="" data-thumb="#thirdFloorUpStairs" position="6.000 2.000 -10.000" rotation="0.000 -15.000 0.000" scale="1.500 1.500 0.000"></a-entity>
+					<a-entity link="href: secondFloor.html;" template="src: #link" data-src="" data-thumb="#secondFloorUpStairs" position="6.000 -2.000 -10.000" rotation="0.000 -15.000 0.000" scale="1.500 1.500 0.000"></a-entity>
```
Comment (Andreybest) [inline index.html:62]: Remove duplicating links

**Review by Andreybest — CHANGES_REQUESTED** [review]
All good!
The only thing, you forgot to change arrow in room 218.
And I requested to rename to 'directorsWaitingRoom.jpg', not to 'director'sWaitingRoom.jpg'.
' is a big no no in file namings.
Fix this issues and Im merging this PR to master.

**Comment by Andreybest** [comment]
All good, thanks for a work!

**Pull request merged**